src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -107,7 +107,7 @@ class WorkspaceInvitePage extends React.Component {
      * @returns {Boolean}
      */
     getShouldShowAlertPrompt() {
-        return _.size(lodashGet(this.props.policy, 'errors', {})) > 0 || this.props.policy.alertMessage.length > 0;
+        return _.size(lodashGet(this.props.policy, 'errors', {})) > 0 || lodashGet(this.props.policy, 'alertMessage.length') > 0;
```

Comment:
```suggestion
        return _.size(lodashGet(this.props.policy, 'errors', {})) > 0 || lodashGet(this.props.policy, 'alertMessage', '').length > 0;
```

NAB, maybe slightly more correct? 

Otherwise we are doing `undefined > 0` which does eval to `false` but seems weird.

Reply:
Hmm, yea this is better.

Reply:
Made a 2-min PR to implement this real quick. 
https://github.com/Expensify/App/pull/5738